fhttp/http_client.go

```diff
@@ -1050,3 +1067,11 @@ func generateReuseThreshold(min int, max int) int {
 
 	return min + rand.Intn(max-min+1) // nolint: gosec // we want fast not crypto
 }
+
+func resolve(hostname string, port string, resolve string) (*net.TCPAddr, error) {
```

Comment:
rename in this function the resolve arg as `override` and/or comment that this will basically resolve either first or 3rd arg if not empty?

Reply:
updated